libcloud/storage/drivers/s3.py

```diff
@@ -69,6 +69,8 @@
 
 S3_RGW_OUTSCALE_DEFAULT_REGION = 'eu-west-2'
 
+S3_RGW_DEFAULT_REGION = 'default'
```

Comment:
I would prefer to put non-s3 related stuff and RGW driver in a separate file / module (e.g. drivers/rgw.py or similar).

This module should only contain base classes and S3 driver related code.